build.rs

```diff
@@ -18,8 +18,8 @@ use std::process::exit;
 include!("src/app.rs");
 
 fn main() {
-    match version_check::is_min_version("1.31.0") {
-        Some((true, _)) => {}
+    match version_check::is_min_version("1.33.0") {
+        Some(true) => {}
         // rustc version too small or can't figure it out
         _ => {
             writeln!(&mut io::stderr(), "'lsd' requires rustc >= 1.31.0").unwrap();
```

Comment:
should we also update the error msg to `1.33.0`?

Reply:
good catch! Fixed